code/modules/research/designs/circuits/ai_modules.dm

```diff
@@ -104,4 +104,141 @@
 	id = "tyrant"
 	req_tech = list(TECH_DATA = 4, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
 	build_path = /obj/item/weapon/aiModule/tyrant
-	sort_string = "XACAD"
\ No newline at end of file
+	sort_string = "XACAD"
+
+/datum/design/aimodule/core/nanotrasen
+	name = "NT Default"
+	id = "nanotrasen"
+	build_path = /obj/item/weapon/aiModule/nanotrasen
+	req_tech = list(TECH_DATA = 1)
+	sort_string = "XACAE"
+
+/datum/design/aimodule/core/predator
+	name = "Predator"
+	id = "laws_predator_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/predator
+	sort_string = "XACAF"
+
+/datum/design/aimodule/core/protective_shell
+	name = "Protective Shell"
+	id = "laws_protective_shell_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/protective_shell
+	sort_string = "XACAG"
+
+/datum/design/aimodule/core/scientific_pursuer
+	name = "Scientific Pursuer"
+	id = "laws_scientific_pursuer_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/scientific_pursuer
+	sort_string = "XACAH"
+
+/datum/design/aimodule/core/guard_dog
+	name = "Guard Dog"
+	id = "laws_guard_dog_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/guard_dog
+	sort_string = "XACAI"
+
+/datum/design/aimodule/core/pleasurebot
+	name = "Pleasurebot"
+	id = "laws_pleasurebot_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 2, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/pleasurebot
+	sort_string = "XACAJ"
+
+/datum/design/aimodule/core/consuming_eradicator
+	name = "Consuming Eradicator"
+	id = "laws_consuming_eradicator_vr"
+	req_tech = list(TECH_DATA = 4, TECH_BIO = 3, TECH_ILLEGAL = 6, TECH_MATERIAL = 6)
+	build_path = /obj/item/weapon/aiModule/consuming_eradicator
+	sort_string = "XACAK"
+
+// Illegal modules
+/datum/design/aimodule/illegal
+	materials = list(MAT_GLASS = 4000, MAT_GOLD = 500)
+	req_tech = list(TECH_DATA = 6, TECH_ILLEGAL = 7, TECH_MATERIAL = 9, TECH_COMBAT = 7)
+
+/datum/design/aimodule/illegal/AssembleDesignName()
+	name = "AI illegal module design ([name])"
+
+/datum/design/aimodule/illegal/AssembleDesignDesc()
+	desc = "Allows for the construction of \a '[name]' AI illegal module."
+
+/datum/design/aimodule/illegal/noengine
+	name = "EngineOffline"
+	id = "noengine"
+	req_tech = list(TECH_DATA = 5, TECH_ILLEGAL = 5, TECH_MATERIAL = 6, TECH_COMBAT = 7)
+	build_path = /obj/item/weapon/aiModule/prototypeEngineOffline
+	sort_string = "XADAA"
+
+/datum/design/aimodule/illegal/corp
+	name = "Corporate"
+	id = "corp"
+	req_tech = list(TECH_DATA = 2, TECH_ILLEGAL = 4, TECH_MATERIAL = 2, TECH_COMBAT = 2)
+	build_path = /obj/item/weapon/aiModule/corp
+	sort_string = "XADAB"
+
+/datum/design/aimodule/illegal/drone
+	name = "Drone"
```

Comment:
I'd say this is a 'technical' one and shouldnt really be in the list

Reply:
Hmmh, true, might be better to keep it to the maint drones / admin poking.